### DISCUSSION TITLE
macOS install location?

### DISCUSSION THREAD

**Discussion by dguion** [microsoft/openjdk#9] [Q&A] [answered]
Hi -

I just downloaded the macOS x64 pkg installer and ran it. I could not find the location of where it puts the binaries. Is there documentation on install location for OSX as well as switching between various jvm versions on OSX? 

Thanks in advance.

**Comment by karianna** [comment, marked as answer]
Hi @dguion - The installer puts it in the 'std' place for Mac OS X (FYI almost all OpenJDK vendors follow this location when using pkg installers). In your terminal window try:

`ls -al /Library/Java/JavaVirtualMachines/`

We recommend using [jenv](https://www.jenv.be/) to manage those versions.

**Reply by dguion** [reply]
ah there it is! thank you. I was looking in `/System/Library/Frameworks/JavaVM.framework/Versions/` I will try using `jenv` to manage versions.

I appreciate your help, thanks again.
